src/devtools/client/debugger/src/reducers/breakpoints.ts

```diff
@@ -371,3 +381,120 @@ export function getLogpointsForSource(state: UIState, sourceId: string) {
   const breakpoints = getBreakpointsList(state);
   return breakpoints.filter(bp => bp.location.sourceId === sourceId).filter(bp => isLogpoint(bp));
 }
+
+export type LocationAnalysisPoints = {
```

Comment:
I'm tempted to call this `LocationAnalysisResult`, since it also includes a potential error, but YMMV.

Reply:
I take it back. I think the nicest API to have here would be:

```
LocationAnalysisStatus {
	data: PointDescription[];
	error: AnalysisError | undefined;
	status: AnalysisStatus; // probably like "Running", "Errored", or "Completed"?
}
```

But maybe that's a follow-on.